src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+
+	"kick_success": "{emote_success} **{target_tag}** `({target_id})` wurde für **{reason}** `[#{infractionId}]` gekickt",
+	"kick_fail": "{emote_warn} Nichtmöglich **{target_tag}** `({target_id})` zu kicken",
+
+	"purge_too_many": "{emote_warn} Ich kann nur 200 Nachrichten gleichzeitig löschen",
+	"purge_too_few": "{emote_warn} Du kannst nicht weniger als eine Nachricht von mir löschen lassen :confused: Warum sollte man sowas überhaupt machen",
+
+	"verification_non_integer": "{emote_warn} Level benötigt zum Integrieren",
+	"verification_community_zero": "{emote_warn} Verifikations Level für Community-Server nicht unter `0` gesetzt werden!",
+	"verification_level_error": "{emote_warn} Das Verifikations Level kann nicht höer als `4` oder niedriger als `0` gesetzt werden",
+	"verification_level_success": "{emote_success} Das Verifikations-Level wurde erfolreich zu  `{level}` geändert",
+
+	"ping_latency": "Der Bot Ping beträgt: **{latency_bot}ms**\nWebSocket Ping ist **{latency_ws}ms**",
+
+	"uptime_uptime": "Der Bot ist seit **{uptime}** online",
+
+	"license_license": "📜 Der Bot ist unter der MIT license lizensiert, Für mehr Informationen und um die volle Lienz lesen zu können, drücke auf Folgenden Link: **[here](https://github.com/TestersQTs/Bulbbot/blob/master/LICENSE)**",
+
+	"privacy_policy": "📜 Lese hier die Datenschutzbestimmungen **[here](https://www.notion.so/Bulbbot-Privacy-Policy-cc22832e352145759d335132a956a252)**",
+
+	"github_source_code": "{emote_github} Der Bot ist komplett Opensource, Du kannst hier den Source-Code Finden: **[here](https://github.com/TestersQTs/Bulbbot)**",
+
+	"invite_desc": "**Lade Bulbbot hier zu deinem Server ein:**\n{bot_invite}\n\n**Support-Server**\n{support_guild}",
+
+	"inviteinfo_error": "{emote_warn} Ich konnte keinen Server mit dem Einladungslink finden",
+
+	"multiban_success": "{emote_success} {full_list} wurden gebannt für **{reason}**",
+
+	"multiunban_success": "{emote_success} {full_list} wurden entbannt für **{reason}**",
+
+	"multikick_success": "{emote_success} {full_list} Wurden gekickt für **{reason}**",
+
+	"multiwarn_success": "{emote_success} {full_list} Wurden verwarnt für **{reason}**",
+
+	"multivoicekick_success": "{emote_success} {full_list} wurden aus dem Voice-Channel für Folgenden Grund Entfernt: **{reason}**",
+
+	"softban_success": "{emote_success} **{target_tag}** `({target_id})` Wurde für Folgenden Grund gebannt: **{reason}** `[#{infractionId}]`",
+
+	"deafen_already_deaf": "{emote_warn} Der Benutzer ist schon Kopfhörer Stummgeschaltet!",
+	"deafen_success": "{emote_success} **{target_tag}** `({target_id})` Kann wegen folgendem Grund nichtsmeht hören: **{reason}** `[#{infractionId}]`",
+
+	"undeafen_not_deaf": "{emote_warn} Benutzer ist nicht Kopfhörer stummgeschaltet!",
+	"undeafen_success": "{emote_success} **{target_tag}** `({target_id})` Kann wegen folgendem Grund wieder hören: **{reason}** `[#{infractionId}]`",
```

Not quite sure on "zuhören" yet
```suggestion
	"undeafen_success": "{emote_success} **{target_tag}** `({target_id})` kann wegen des folgenden Grundes wieder zuhören: **{reason}** `[#{infractionId}]`",
```